Mathlib/Algebra/Order/AbsoluteValue/Basic.lean

```diff
@@ -351,6 +351,17 @@ lemma not_isNontrivial_apply {v : AbsoluteValue R S} (hv : ¬ v.IsNontrivial) {x
     v x = 1 :=
   v.not_isNontrivial_iff.mp hv _ hx
 
+omit [IsOrderedRing S] in
```

Comment:
I am not really sure that these two lemmas are needed: their proofs are one-liner, and it is unclear to me when to stop...